src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/ExpressionTreeCallRewriter.cs

```diff
@@ -241,7 +241,7 @@ private Expr GenerateLambda(ExprCall pExpr)
         /////////////////////////////////////////////////////////////////////////////////
 
         [RequiresUnreferencedCode(Binder.TrimmerWarning)]
-        private Expression GenerateCall(ExprCall pExpr)
+        private MethodCallExpression GenerateCall(ExprCall pExpr)
```

Comment:
@cston, should we make changes like this to Microsoft.CSharp and System.Linq.Expressions, or should we suppress the analyzer for these libs?

Reply:
Let's suppress the analyzer for Microsoft.CSharp and System.Linq.Expressions and revert the source changes to those libraries, thanks. cc @jaredpar

Reply:
I understand Microsoft.CSharp but why also System.Linq.Expressions?

Reply:
For both libraries, we're limiting the set of changes to reduce the risk of regressions (see [README.md](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Linq.Expressions/README.md)).